spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/KubernetesInformerDiscoveryClient.java

```diff
@@ -105,24 +124,22 @@ public String description() {
 	public List<ServiceInstance> getInstances(String serviceId) {
 		Objects.requireNonNull(serviceId, "serviceId must be provided");
 
```

Comment:
because we now have a list of listers for services/endpoints the code in the client needs to be slightly adjusted, also a little bit simplified here and there, like from:

```
if (services.size() == 0 || !services.stream().anyMatch(service -> matchesServiceLabels(service, properties))) {
```

to:

```
if (services.size() == 0 || services.stream().noneMatch(service -> matchesServiceLabels(service, properties))) {
```